doc/api/stream.md

```diff
@@ -277,7 +277,8 @@ added: v0.9.4
 The `'error'` event is emitted if an error occurred while writing or piping
 data. The listener callback is passed a single `Error` argument when called.
 
-The stream is not closed when the `'error'` event is emitted.
+The stream is not closed when the `'error'` event is emitted unless the
+[`autoDestroy`][] option was set to `true` when creating the stream.
```

Comment:
I'm not sure how to create a reference here?

Reply:
Maybe something like this:
```suggestion
[`autoDestroy`][writable-new] option was set to `true` when creating the stream.
```
with adding this references between these lines in the bottom:
```diff
[writable-destroy]: #stream_writable_destroy_error
+[writable-new]: #stream_constructor_new_stream_writable_options
[zlib]: zlib.html
```